src/util/mqtt_config.ts

```
@@ -48,5 +48,16 @@ export function convertConfigForMqttOptions(config: Mqtt) {
     options.rejectUnauthorized = false;
   }
 
+  if (config.base_topic === undefined) {
+    config.base_topic = 'p2m';
+  }
+
+  options.will = {
+    topic: config.base_topic + '/bridge_availability',
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)